macros/distributions/discrete/probabilities.sql

```diff
@@ -18,9 +18,9 @@
         {{ exceptions.raise_compiler_error("`probabilities` must sum to 1.0, not " + ns.values|sum|string) }}
     {%- endif -%}
 
-    {%- if ns.keys[0] is number -%}
+    {%- if ns.keys[0] is number or type!="string" -%}
```

Comment:
what would happen if `ns.keys[0] is string` but `type == "number"` or `type == "banana"`?

Is it right to treat those both as number with `wrap = ""`? will that break anything?